mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -267,8 +265,8 @@ public static boolean hiveCatalog(Configuration conf, Properties props) {
    */
   private static String getCatalogType(Configuration conf, String catalogName) {
     if (catalogName != null) {
-      String catalogType = conf.get(String.format(InputFormatConfig.CATALOG_TYPE_TEMPLATE, catalogName));
-      if (catalogName.equals(ICEBERG_HADOOP_TABLE_NAME) || catalogType == null) {
+      String catalogType = conf.get(InputFormatConfig.catalogTypeConfigKey(catalogName));
+      if (catalogName.equals(ICEBERG_HADOOP_TABLE_NAME)) {
```

Comment:
Do we need to update the javadocs of this method according to this change?

Reply:
updated

Reply:
I moved the doc of this method to `Catalogs` class level, because private method docs are not generated in html.